src/components/animation-app.tsx

```diff
@@ -45,18 +56,26 @@ export const AnimationApp = ({ ac }: AnimationAppProps) => {
             ac={ac}
             playing={playingTissue}
             setPlaying={setPlayingTissue}
-            title={ac.o("LEFTANIMATIONTITLE")}
```

Comment:
Do you want `disabled={playingCell}` on the tissue video? Right now behaviors don't match.

Reply:
It's weird, but it's intended that they work asymmetrically. We'll see what the PIs think once they realize how things actually work based on their design.